README.md

```
@@ -1,17 +1,20 @@
-## Usage
-
-```
+WS2WH is a lightweight bridge that connects WebSocket clients to HTTP webhook endpoints. It enables real-time,
+bidirectional communication by converting WebSocket messages into HTTP POST requests and vice versa. This tool is
+particularly useful when you need to integrate WebSocket-based clients with HTTP-only backend services, or when you
+want to add WebSocket support to existing HTTP APIs without modifying the backend. With a simple configuration, WS2WH
+handles the protocol translation and message routing, making it an ideal solution for scenarios requiring real-time
+updates in HTTP-based architectures.
+
+## Usage```shell
 ws2wh -b https://example.com/api/v1/webhook -r /reply -l :3000 -p / -v INFO -h localhost
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)